.github/workflows/notify.yml

```diff
@@ -0,0 +1,18 @@
+name: Notify Antora
+
+on:
+  push:
+    branches:
+      - publish
+
+jobs:
+  notify:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Notify Antora
+        uses: peter-evans/repository-dispatch@master
+        with:
+          token: ${{ secrets.BUILD_ACCESS_TOKEN }}
```

Comment:
@adam-cowley @jexp could you please add this secret?

Reply:
I've just added the secret across the neo4j-documentation organisation